From febe9530989d970c1391d4cbb2a85875d0146273 Mon Sep 17 00:00:00 2001
From: Mirza Krak <mirza.krak@northern.tech>
Date: Mon, 27 Aug 2018 14:42:10 +0200
Subject: [PATCH 1/2] env: Kconfig: remove default ENV_OFFSET for SUNXI

Mender build will set this up correctly in meta-mender

Signed-off-by: Mirza Krak <mirza.krak@northern.tech>
---
 env/Kconfig | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/env/Kconfig b/env/Kconfig
index 8c9d800f48..020cdaf9b4 100644
--- a/env/Kconfig
+++ b/env/Kconfig
@@ -406,14 +406,6 @@ config ENV_FAT_FILE
 
 if ARCH_SUNXI
 
-config ENV_OFFSET
-	hex "Environment Offset"
-	depends on !ENV_IS_IN_UBI
-	depends on !ENV_IS_NOWHERE
-	default 0x88000 if ARCH_SUNXI
-	help
-	  Offset from the start of the device (or partition)
-
 config ENV_SIZE
 	hex "Environment Size"
 	depends on !ENV_IS_NOWHERE
-- 
2.18.0

